lib/cli/index.js

```diff
@@ -11,19 +11,18 @@ const reports = fs.readJsonSync(config.reportsFile.JSON);
  * Run lib
  */
 async function run() {
-    const { testRunKey, target } = opts;
+    const { testRunKey, target, noUpdateCases } = opts;
     const testHandler = factory.create(target);
 
     if (testHandler === null) throw Error('Could not determine testHandler');
     const testResults = testHandler.getTestResults(reports);
-    const result = testHandler.formingScripts(testResults);
+    const result = await testHandler.formingScripts(testResults);
     const resultTestCase = testHandler.formingSteps(testResults);
 
+    if (!noUpdateCases) await testCase.prepare(resultTestCase);
```

Comment:
I think it needs to be changed
I think we should use the option to update the cases, we should not use negation